lib/handlers/alias.get.js

```diff
@@ -1,42 +1,51 @@
 'use strict';
 
-const HttpError = require('http-errors');
 const { validators } = require('@asset-pipe/common');
+const HttpError = require('http-errors');
+const abslog = require('abslog');
 const HttpOutgoing = require('../classes/http-outgoing');
 const Alias = require('../classes/alias');
 const utils = require('../utils/utils');
 
-const handler = async (sink, req, org, type, name, alias, extra) => {
-    try {
-        validators.alias(alias);
-        validators.extra(extra);
-        validators.name(name);
-        validators.type(type);
-        validators.org(org);
-    } catch (error) {
-        throw new HttpError(404, 'Not found');
+class AliasGet {
+    constructor(sink, config = {}, logger) {
```

Comment:
Is there a reason to have logger as a separate arg and not part of config?

Reply:
I am thinking that config is something which is passed in from a config system like convict or something. If we add sink and logger to the config object it becomes a fair amount of object composition etc so I found it easier to keep them separate arguments.